src/task-actions.ts

```diff
@@ -594,3 +594,11 @@ export async function starknetRunAction(
         await runSuper(args);
     });
 }
+
+export async function starknetPluginVersionAction(
+) {
+    const version = JSON.parse(
```

Comment:
Can you consider something like this: https://stackoverflow.com/questions/9153571/is-there-a-way-to-get-version-from-package-json-in-nodejs-code

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/181/commits/30653a3ad9cfab02eaf9139abe1df9471489fcc7